.pre-commit-config.yaml

```diff
@@ -99,6 +96,11 @@ repos:
             |^pandas/conftest\.py  # keep excluded
         args: [--disable=all, --enable=redefined-outer-name]
         stages: [manual]
+    -       id: pylint
+            alias: unspecified-encoding
+            name: Using open without explicitly specifying an encoding
+            args: [--disable=all, --enable=unspecified-encoding]
+            stages: [manual]
```

Comment:
could you keep the alignment that there is in the rest of the file?

Reply:
i think alignment are perfect for rest of the file

Reply:
pre-commit run unspecified-encoding --hook-stage manual which was skipped

Reply:
```suggestion
    -   id: pylint
        alias: unspecified-encoding
        name: Using open without explicitly specifying an encoding
        args: [--disable=all, --enable=unspecified-encoding]
        stages: [manual]
```

Reply:
i'll do it.

Reply:
is there are no need of b/.pre-commit-config.yaml this file??